packages/core/admin/_internal/node/develop.ts

```diff
@@ -24,6 +27,40 @@ interface DevelopOptions extends CLIContext {
   watchAdmin?: boolean;
 }
 
+const cleanupDistDirectory = async ({
```

Comment:
Might be worth writing a comment explaining this doesn't clean the dist directory, just all files excluding those in the admin build folder?

Reply:
Done